client/src/components/Timeline.vue

```diff
@@ -69,6 +69,10 @@ export default {
           display: false
         },
         scales: {
+          yAxes: [{
+            beginAtZero: false
```

Comment:
Won't this change the behaviour of all the charts that use Timeline.vue? Is that what we want?

Reply:
@Fingel This was addressed in the second commit -- I added exceptions for certain units.